### PR TITLE
chore (styles): modal overlay color refactor

### PR DESCRIPTION
**Pull request by rlr**
tiny little refactor to make onboarding theming easier.

### CODE REVIEW

File: system-addon/content-src/styles/_theme.scss [hunk 1]
```diff
@@ -89,6 +89,7 @@ body {
   // Modal + overlay
   --newtab-modal-color: $grey-80;
   --newtab-overlay-color: $grey-90;
+  --newtab-overlay-color: rgba($grey-90, 0.8);
```
Comment (piatra) [inline system-addon/content-src/styles/_theme.scss]: Does the previous definition need to get removed?

Comment (rlr) [reply system-addon/content-src/styles/_theme.scss]: uhm. yes. indeed.

**Comment by piatra** [comment]
We have a style in `_variables.scss` where an opacity modifier adds an extra `-<opacity-value>` to the variable name. Should we follow the same pattern? I.e. `$grey-90-80` is already defined for example.

**Comment by rlr** [comment]
there was also a lint warning I just fixed.

**Comment by rlr** [comment]
you are right, sir. thank you!

**Comment by as-pine-proxy** [comment]
[Treeherder: 01a07aca086cbbffadc23548a6ea9942e86e75d4](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=01a07aca086cbbffadc23548a6ea9942e86e75d4)

**Pull request merged**